src/App.jsx

```diff
@@ -14,45 +16,115 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FILED_ALPHABETICALLY = 'alphabetically';
+const SORT_FILED_BY_LENGTH = 'length';
+const REVERSE = true;
```

Comment:
will be better to rename to
```suggestion
const REVERSE_DEFAULT_STATE = true;
```